packages/front-end/components/Features/ConditionDisplay.tsx

```diff
@@ -57,11 +66,20 @@ function needsValue(operator: string) {
 function getValue(
   operator: string,
   value: string,
+  field: string,
```

Comment:
same thing here. if I can build a new operator $beforeDate and $afterDate I can key it off that instead.